index.html

```diff
@@ -0,0 +1,131 @@
+                            us </p>
+                    </div>
+                </div>
+                <div class="card00">
+                    <div class="icon icon2"><img src="./images/art.png" alt=""></div>
+                    <div class="card-content">
+                        <h3>art</h3>
+                        <p>Are you a designer ? contact our platform and let's together make it happen, make this space
+                            greater</p>
+                    </div>
+                </div>
+                <div class="card00 icon3">
+                    <div class="icon"><img src="./images/binoculars.png" alt=""></div>
+                    <div class="card-content">
+                        <h3>future</h3>
+                        <p>NFTs are digital birth certificates, let's slide throught the time, let's make it our future
+                        </p>
+                    </div>
+                </div>
+                <div class="card00">
+                    <div class="icon"><img src="./images/financial-growth.png" alt=""></div>
+                    <div class="card-content">
+                        <h3>financial-growth</h3>
+                        <p>Anyone who follows the crypto space knows that non-fungible tokens (NFTs) are all the rage of
+                            late.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="collabotators">
+            <h2>Articles</h2>
+            <div class="collabo-container">
+
+            </div>
+            <div class="see-more">SEE MORE</div>
+            <div class="see-less">SEE LESS</div>
+        </div>
+    </main>
+
+    <footer>
+        <h2 class="footer-line-1">Partners</h2>
+        <div class="footer-list-container">
+            <div class="for-nav"><img src="./images/polygonlogo.jpg" alt="Here is the polygon company logo"></div>
+            <div class="for-nav"><img src="./images/NflLogo.png" alt="Here is the Nfl company logo"></div>
+            <div class="for-nav"><img src="./images/NBA.jpg" alt="Here is the NBA company logo"></div>
+            <div class="for-nav"><img src="./images/ufc.jpg" alt="Here is the ufc company logo"></div>
+            <div class="for-nav"><img src="./images/coinbase.png" alt=""></div>
+        </div>
+
+        <ul class="footer-list">
+            <li>Polygon</li>
+            <li>NFL</li>
+            <li>NBA</li>
+            <li>UFC</li>
+            <li>coinbase</li>
+            <li>Fantomas</li>
+            <li>Micielo Hills</li>
+        </ul>
+
+        <div class="bottom">
+            <div class="bottom-img"><img src="./images/NftLogo.jpg" alt=""></div>
+            <div class="bottom-content">
+                <img src="./images/copyright.webp" alt="">
+                <P>Published by Micielo Hills company <br> copyright November 05 2020</P>
+            </div>
+        </div>
```

Comment:
- [ ] I suggest that you add some left and right padding to the `.bottom` class to give the footer some breathing space as seen the original design. Also try to align the child items to the middle or center